ts/features/idpay/initiative/configuration/screens/InstrumentsEnrollmentScreen.tsx

```diff
@@ -55,9 +58,17 @@ const InstrumentsEnrollmentScreen = () => {
     return instrumentItemsRef.current;
   };
 
-  const selectedInstrumentIdRef = useRef<number | undefined>(undefined);
+  const selectedInstrumentIdRef = React.useRef<number | undefined>(undefined);
 
   const isLoading = useSelector(configurationMachine, isLoadingSelector);
+  const initiativeDetails = useSelector(
```

Comment:
What do you think about using useIOSelector here?

Reply:
`useIOSelector` is a typed selector for the redux store, in this case `useSelector` is exported from `@xstate/react` and it's used to get data from XState context :)